com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -1618,8 +1618,8 @@ class TemporalAntiAliasingData
                 passData.temporalAAMaterial.EnableKeyword("ENABLE_MV_REJECTION");
             }
 
-            passData.runsTAAU = TAAU;
-            if (TAAU && !postDoF)
+            passData.runsTAAU = upscalingEnabled;
+            if (upscalingEnabled && !postDoF)
```

Comment:
This (and I suspect other of these changes) is wrong. This will flag TAA Upscale mode not necessarily when TAAU is on. I would keep, at the very least for clarity, the checks that are explicitly needed only for TAAU separate.